Makefile

```diff
@@ -128,7 +128,7 @@ $(addprefix collect-templates-,$(SUPPORTED_VERSIONS)): collect-templates-%:
 	$(eval version:=$*)
 	mkdir -p ${TEMPLATES_OUT_DIR}/v${version}
 	cp ${RESOURCES_DIR}/smcp-templates/v${version}/${BUILD_TYPE} ${TEMPLATES_OUT_DIR}/v${version}/default
-	find ${RESOURCES_DIR}/smcp-templates/v${version}/ -maxdepth 1 -type f ! -name "maistra" ! -name "servicemesh" -exec cp {} ${TEMPLATES_OUT_DIR}/v${version}/ \;
+	find ${RESOURCES_DIR}/smcp-templates/v${version}/ -maxdepth 1 -type f ! -name "maistra" ! -name "servicemesh" | xargs cp -t ${TEMPLATES_OUT_DIR}/v${version}
```

Comment:
Why are we changing this? I believe @fjglira recently replaced the `| xargs` with `-exec` so that the build would work on MacOS.

Reply:
Yes, please do not change this again because will break the build on macOS. We need to keeped like this:
`find ${RESOURCES_DIR}/smcp-templates/v${version}/ -maxdepth 1 -type f ! -name "maistra" ! -name "servicemesh" -exec cp {} ${TEMPLATES_OUT_DIR}/v${version}/ \;`

Reply:
@luksa @fjglira This was automatically changed with `make gen` so does that mean that there needs to be changes else where to ensure it is not changed again while running `make gen`?

Reply:
I do not know, I was running also make gen but from macOS. I do not know if has anything to do